AI/Nullkiller/Pathfinding/AINodeStorage.cpp

```diff
@@ -1060,6 +1056,55 @@ struct TowmPortalFinder
 	}
 };
 
+template<class T>
```

Comment:
Could you please rename the template parameter to `Vector` to make it more explicit that it's a vector?

```suggestion
template<class Vector>
```